### PR TITLE
FEATURE: Add python example aws eks

### PR DESCRIPTION
**Pull request by smalltown**
Add one python example to demonstrating below CDK for Terraform features:

- TerraformOutput
- Data
- Tokens

### CODE REVIEW

File: examples/python/aws-eks/Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+constructs="*"
+cdktf="*"
```
Comment (skorfmann) [inline examples/python/aws-eks/Pipfile:11]: Please use the relative reference here `cdktf = {path = "./../../../dist/python/cdktf-0.0.0-py3-none-any.whl"}`.

File: examples/python/aws-eks/Pipfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+constructs="*"
```
Comment (skorfmann) [inline examples/python/aws-eks/Pipfile:10]: I think this should be pinned to version 3

File: examples/python/aws-eks/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+dist/
+imports/
+terraform.tfstate*
```
Comment (skorfmann) [inline examples/python/aws-eks/.gitignore:3]: Adding `cdktf.out` would be good, I think.

**Comment by smalltown** [comment]
@skorfmann Thank for your review very much ^^
I have push another commit to fix it！

**Comment by smalltown** [comment]
@skorfmann Thank for your review very much ^^
I have pushed another commit to fix it !!

File: examples/python/aws-eks/main.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+class MyStack(TerraformStack):
+    def __init__(self, scope: Construct, ns: str):
+        super().__init__(scope, ns)
+
+        AwsProvider(self, 'Aws', region='us-west-2')
+        LocalProvider(self, 'Local')
+        NullProvider(self, 'Null')
+        RandomProvider(self, 'Random')
+        TemplateProvider(self, 'Template')
```
Comment (cmclaughlin) [inline examples/python/aws-eks/main.py:26]: Perhaps this is more of a Terraform question and less of a cdktf question, but do the last four providers do anything?

Comment (skorfmann) [reply examples/python/aws-eks/main.py:26]: Hm, not sure - I think these providers would be pulled in automatically if needed - aren't they?

Comment (smalltown) [reply examples/python/aws-eks/main.py]: Hi @cmclaughlin and @skorfmann,

Thank for your reviewing,
Yes, the four providers not real take any effect during develop cdk for terraform,
and the provider will be downloaded when user perform `terraform init`,
hence, I have removed the four provider declaration and including from the example.

**Review by skorfmann — APPROVED** [review]
Awesome, thanks again 👍

**Pull request merged**